README.md

```diff
@@ -190,6 +190,15 @@ function onAssemblyProgress(assembly) {
 }
 ```
 
+**Tip:** `createAssembly` returns a `Promise` with an extra property `assemblyId`. This can be used to retrieve the assemblyId before the assembly has even been created. Useful for debugging by logging this ID when the request starts, for example:
```

Comment:
```suggestion
**Tip:** `createAssembly` returns a `Promise` with an extra property `assemblyId`. This can be used to retrieve the Assembly ID before the Assembly has even been created. Useful for debugging by logging this ID when the request starts, for example:
```